src/editors/containers/VideoEditor/components/VideoSettingsModal/components/LicenseWidget/LicenseSelection.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { connect, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+  injectIntl,
+  intlShape,
+} from '@edx/frontend-platform/i18n';
+import {
+  Form,
+  Icon,
+  IconButtonWithTooltip,
+} from '@edx/paragon';
+import { Delete } from '@edx/paragon/icons';
+
+import { actions, selectors } from '../../../../../../data/redux';
+import hooks from './hooks';
+import messages from './messages';
+import { LicenseLevel, LicenseNames, LicenseTypes } from '../../../../../../data/constants/licenses';
+
+/**
+ * Collapsible Form widget controlling videe license type and details
+ */
+export const LicenseSelection = ({
```

Comment:
nit: The name here doesn't really capture what is does? LicenseSelector or LiscenseDropdown might be good options for names